src/scss/_functions.scss

```diff
@@ -3,8 +3,27 @@
 /// @param {Number} $index - number of the scale to return
 ///
 /// @returns {List} list of the requested scale value
-@function oTypographyGetScale($index: 0) {
-	@return map-get($o-typography-font-scale, $index);
+@function oTypographyGetScale($index: 0, $font: '') {
+	$scale: map-get($o-typography-font-scale-by-font, _oTypographyRemoveFontFallback($font));
+	$scale: if(type-of($scale) == 'map', $scale, $o-typography-font-scale);
+	@return map-get($scale, $index);
+}
+
+/// Removes a fonts fallbacks.
+/// @todo move into o-fonts
+///
+/// @param {String} $font - font family
+///
+/// @returns {String} font family without fallbacks
+@function _oTypographyRemoveFontFallback($font) {
+	@if type-of($font) == 'list' {
+		@return nth($font, 1);
+	}
+	$index: str-index($font, ',');
```

Comment:
Would this handle a case if someone _didn't_ put a comma in a list?
Why not use [`index()`](http://sass-lang.com/documentation/Sass/Script/Functions.html#index-instance_method) ?

Reply:
The `font-family` css property should be separated by commas so I'm not worried about that. `index` accepts a list and wouldn't workout with a string:
```
$example: str-index('hello hi', 'hi'); // 7
$example: index('hello hi', 'hi'); // null
$example: index(hello hi, 'hi'); // 2
```

Reply:
what about `index('hello' 'hi', 'hi') //2